api/src/test/java/marquez/MarquezAppIntegrationTest.java

```diff
@@ -447,7 +448,7 @@ public void testApp_createJobAndMarkRunAsComplete() {
     assertThat(jobWithNewVersion.getLatestRun().get().getId()).isEqualTo(runStarted.getId());
 
     // (9) Complete a run
-    final Instant endedAt = newTimestamp();
+    final Instant endedAt = newTimestamp().truncatedTo(ChronoUnit.MICROS);
     final Run runCompleted = client.markRunAsCompleted(run.getId(), endedAt);
```

Comment:
@ucg8j Nice! Though this does seem odd that you'd have to modify the timestamps, but I think it's ok to merge this PR as is and look into this a bit deeper as follow up work.